examples/snapshot_example/snapshot-config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "micro_file_name": "micro_dummy",
+    "snapshot_params": {
+        "parameter_file_name": "parameter.hdf5",
+        "read_data_names": {"macro-scalar-data": "scalar", "macro-vector-data": "vector"},
+        "write_data_names": {"micro-scalar-data": "scalar", "micro-vector-data": "vector"},
+        "postprocessing": "True",
```

Comment:
Instead of having a boolean to activate post-processing and post-processing as part of the micro-simulation class, we want to provide the name of a separate post-processing script. This allows removing the snapshot example and reusing the Micro Manager example micro simulation classes by adding the files necessary to run snapshot computation.

Reply:
Good idea! Let us try this out after the feature itself is close to being merged.